.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version: "1.20"
+          go-version: "1.23"
```

Comment:
If it was still 1.20, the go install failed.

The `shfmt` necessary to be 1.22 or higher.
https://github.com/mvdan/sh?tab=readme-ov-file#sh

I updated it to the latest version, 1.23.